src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -227,10 +228,10 @@ def test_create_advanced_arb_sequence():
         loop_counts_array = [10, 20, 30]
         session.output_mode = nifgen.OutputMode.SEQ
         # Test relies on value of sequence handles starting at a known value and incrementing sequentially. Hardly ideal.
-        assert ([], seq_handle_base + 0) == session.create_advanced_arb_sequence(waveform_handles_array, loop_counts_array=loop_counts_array)
-        assert ([], seq_handle_base + 1) == session.create_advanced_arb_sequence(waveform_handles_array, loop_counts_array=loop_counts_array, sample_counts_array=sample_counts_array)
-        assert (marker_location_array, seq_handle_base + 2) == session.create_advanced_arb_sequence(waveform_handles_array, loop_counts_array=loop_counts_array, marker_location_array=marker_location_array)
-        assert (marker_location_array, seq_handle_base + 3) == session.create_advanced_arb_sequence(waveform_handles_array, loop_counts_array=loop_counts_array, sample_counts_array=sample_counts_array, marker_location_array=marker_location_array)
+        assert (array.array('l'), seq_handle_base + 0) == session.create_advanced_arb_sequence(waveform_handles_array, loop_counts_array=loop_counts_array)
```

Comment:
I must say this is a little strange. What does documentation say this returns?
http://nimi-python.readthedocs.io/en/master/nifgen/functions.html#nifgen.Session.create_advanced_arb_sequence

```
coerced_markers_array (list of int):

Returns an array of all given markers that are coerced (rounded) to the nearest marker quantum. Not all devices coerce markers.

Default Value: None
```

Reply:
Fixed documentation. Now says `coerced_markers_array (array.array("l")):`